ui/lib/core/addon/components/field-group-show.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @module FieldGroupShow
+ * FieldGroupShow components are used to...
+ *
+ * @example
+ * ```js
+ * <FieldGroupShow @param1={param1} @param2={param2} />
```

Comment:
We should update these docs (and eventually in another PR add to Storybook). 😬

Reply:
Yeah there's lots to add to storybook because we moved a bunch over to `core`.